entity_version.post_update.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ * Entity version post updates.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Apply entity version settings for bundles with version field.
+ */
+function entity_version_post_update_configure_settings() {
+  // Introduced new configuration to select a version field of an entity bundle
+  // as the main version field to use by the (sub)module functionalities,
+  // instead of searching for all version fields on the bundle and using the
+  // first one.
+  $versioned_entity_types = \Drupal::service('entity_field.manager')->getFieldMapByFieldType('entity_version');
```

Comment:
With the previous approach, we were using `\Drupal::service('entity_field.manager')->getFieldDefinitions(<entity>, <bundle>)`.
These two methods return different results. The field map returns the fields ordered by creation time, while the other returns the field alphabetically.
So if you checkout a previous release and create one field called `field_z` and then a field called `field_a`.
Test for example the workflow on it. You will get changes in the `field_a` field (first alphabetical).
Now checkout this branch and test the update. The configuration will be created with using the `field_z` (first chronological). This is because those two methods use different ways of determining the fields.
We need to create the new configurations by using the same method in the past.

Reply:
With the previous approach => With the past release of the module.